Rytis_George_Baltakys/rails/posts_comments/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PostsController < ApplicationController
+
+	def index
+		@posts = Post.all.includes(:user)
+	end
+
+	def create
+		post = Post.new(post_params.merge(user: User.find(session[:user_id])))
+		if post.valid?
```

Comment:
post.save calls valid in the chain of methods, so just calling post.save and based on true or false from that does the same thing. Technically are calling .valid? twice.